README.md

```diff
@@ -9,6 +9,40 @@ From time to time there is a need to merge the changes from the original repo in
 | --- | --- |
 |**Original corresponding commit hash:**| `d50aef95520df4216c638495a6049125c00742cb` |
```

Comment:
What does this refer to ?

Reply:
this refers to the hash of the **newest** commit authored `FreeRdp/FreeRdp` which is the parent of the **oldest** commit authored by `UiPath/FreeRdp`